build.gradle.kts

```diff
@@ -7,6 +7,7 @@ plugins {
     id("org.springframework.boot") version "2.7.0" apply false
     id("io.spring.dependency-management") version "1.0.11.RELEASE" apply false
     id("org.springframework.cloud.contract") version "3.1.1" apply false
+    id("au.com.dius.pact") version "4.3.10" apply false
```

Comment:
Currently, the project uses version `4.3.6` of the PACT libraries. Maybe we should update those to the same version?

Reply:
done